content/sql/concepts/commands/terms/create-view/create-view.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'CREATE VIEW'
+Description: 'Creates a persistent virtual table based on a saved query.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'SQLite'
+  - 'Tables'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **`CREATE VIEW`** command creates a persistent virtual table based on a saved query. The resulting view can be accessed via a [`SELECT`](https://www.codecademy.com/resources/docs/sql/commands/select) statement, just like a normal table in the database.
```

Comment:
I'm not sure about using the word "persistent" here. It may be confused with persisted computed columns. 

```suggestion
The **`CREATE VIEW`** command creates a virtual table based on a saved query. The resulting view can be accessed via a [`SELECT`](https://www.codecademy.com/resources/docs/sql/commands/select) statement, just like a normal table in the database.
```